### PR TITLE
Tweak roadmap

### PR DESCRIPTION
**Pull request by stefanalund**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: docs/overview/roadmap.md [hunk 1]
```diff
@@ -5,13 +5,11 @@ title: Project roadmap
...
-> Backstage is currently in Alpha. This means that you can expect APIs and
-> features to change until we reach our
-> [Backstage Beta milestone](https://github.com/spotify/backstage/milestone/19).
-> Before we have API stability, plugins will need ongoing maintenance. It is
-> also recommended that teams who adopt Backstage today upgrades their
-> installation as new [releases](https://github.com/spotify/backstage/releases)
-> are available.
+> Backstage is currently under rapid development. This means that you can expect
+> APIs and features to evolve. It is also recommended that teams who adopt
+> Backstage today upgrades their installation as new
```
Comment (freben) [inline docs/overview/roadmap.md:10]: `upgrade` (because "teams")

File: docs/overview/roadmap.md [hunk 2]
```diff
@@ -5,13 +5,11 @@ title: Project roadmap
...
+> [releases](https://github.com/spotify/backstage/releases) are available, as
```
Comment (freben) [inline docs/overview/roadmap.md:11]: `become available`

File: docs/overview/roadmap.md [hunk 3]
```diff
@@ -83,9 +81,9 @@ guidelines to get started.
 
 ### Future work 🔮
 
-- **[Backstage Beta](https://github.com/spotify/backstage/milestone/19)** - The
-  platform APIs and features are stable and can be dependent on for production
-  use. After this plugins will require little to no maintenance.
+- **[Backstage platform is stable](https://github.com/spotify/backstage/milestone/19)** -
+  The platform APIs and features are stable and can be dependent on for
```
Comment (freben) [inline docs/overview/roadmap.md:85]: `depended`

**Review by freben — APPROVED** [review]

**Comment by stefanalund** [comment]
@freben 👍 fixed.

**Pull request merged**